### ISSUE TITLE
Incorrect reference to original file given for FileReference

### ISSUE THREAD

**Issue by lapierrec** [RobertBecker/iri#1]
Hi, if I create an Interchange Responsive Image content, I've no problem with saving. But, when I choose a media in source, single or multiple, I have an error message and the media was not recorded.  This is the error message :

#1300098528: Incorrect reference to original file given for FileReference. ([More information](https://wiki.typo3.org/Exception/CMS/1300098528))
InvalidArgumentException thrown in file
/var/www/typo3_src-8.7.11/typo3/sysext/core/Classes/Resource/FileReference.php in line 86.

I cleared the cache in InstallTool and compared the database. Thanks !

**Comment by RobertBecker** [comment]
Hi, thank you for your feedback. 

The error above is a known issue. You can prevent this failure by saving a new source record before you fill it. After the reload it works as expected.

**Comment by lapierrec** [comment]
I tried with a empty source. I saved it, but when I return to fill it and save, I have always same error message. I checked in Installed Extensions. I see update script's button, but when I click I don't get the update. Just these messages.
![iri_messages](https://user-images.githubusercontent.com/18560613/38108676-925d2bbe-3363-11e8-9b14-e77f237a8ab3.png)

**Comment by RobertBecker** [comment]
There is nothing to update. The tables are fine. 

Have you already created the queries like described in the Administrator Manual (https://docs.typo3.org/typo3cms/extensions/iri/AdministratorManual/Index.html)?

**Comment by lapierrec** [comment]
Yes, I made 3 Queries with different values. I have a supplement input than your screenshot : cropping_variant. It's a drop-down menu with Mobile, Tablet and Desktop values.

**Comment by RobertBecker** [comment]
I reproduced the error and provided the bugfix release [2.0.2](https://github.com/RobertBecker/iri/releases/tag/2.0.2).

**Comment by lapierrec** [comment]
Great! Thanks for the release. But, I've a error message when I import the new version in the extension module. 
<img width="938" alt="error_extension_module" src="https://user-images.githubusercontent.com/18560613/38691760-c143216c-3e4f-11e8-99b0-63f1b3ab1744.png">

**Comment by lapierrec** [comment]
I changed my TYPO3 version 8.7.16, I was on 8.7.11, and now, I can add this extension. But I always have a SQL error when I try to add something like modes. Sometime it's starttime, sometime language.
`2: SQL error: 'Incorrect integer value:' 'for column' starttime 'at row 1' (tx_iri_domain_model_mode: NEW5b338f51d5c55349687586)`
Thanks a lot!
